evap/staff/templates/staff_questionnaire_index_list.html

```diff
@@ -25,7 +25,7 @@ <h3 class="panel-title">{% trans headline %}</h3>
                             {% blocktrans count questionnaire.contributions.count as count %}used {{ count }} time {% plural %} used {{ count }} times {% endblocktrans %}
                         </td>
                         <td class="right">
-                            {% if not questionnaire.obsolete %}
```

Comment:
so previously obsolete questionnaires could not be edited. now, as used questionnaires cannot be edited, i thought that wouldn't matter anymore. @janno42 please speak up if you think otherwise.